src/engine/source/api/src/catalog/catalog.cpp

```diff
@@ -246,6 +250,8 @@ std::optional<base::Error> Catalog::postResource(const Resource& collection, con
 
 std::optional<base::Error> Catalog::putResource(const Resource& item, const std::string& content)
 {
+    using Format = ::com::wazuh::api::engine::catalog::ResourceFormat;
```

Comment:
Could it be that this resource is not being used?

Reply:
good catch!